app/controllers/account.controller.ts

```diff
@@ -0,0 +1,68 @@
+import { createAccountValidation } from '../middlewares/validation.js';
+import { Request, Response } from 'express';
+import { errorResponse, successResponse } from '../utils/responseHandler.js';
+import {
+  createNewAccount,
+  getAllAccounts,
+} from '../services/account.service.js';
+
+export const createAccountController = async (
+  request: Request | any,
+  response: Response
+) => {
+  try {
+    const { error } = createAccountValidation(request.body);
+
+    if (error) return errorResponse(response, error.details[0].message, 400);
+
+    const user = request.user;
+
+    const { balance, isDefault } = request.body;
```

Comment:
You should not be able to create a default account via API, you already create the default account when a user signup correct?